about.html

```diff
@@ -0,0 +1,97 @@
+              <li class="close_menu"> <img src="./src/images/close_menu.png" alt="Close"/></li>
+              <li class="itemProgram"> <a href="#mainProgram">Program</a> </li>
+              <li class="itemSpeaker"> <a href="#speaker">Speaker</a> </li>
+              <li class="itemAbout"> <a href="about.html">About</a> </li>
+              <li class="itemAbout"> <a href="index.html">Home</a> </li>
+            </ul>
+            <ul class="itemsDesktopMenu">
+              <li class="itemProgram"> <a href="#mainProgram">Program</a> </li>
+              <li class="itemSpeaker"> <a href="#speaker">Speaker</a> </li>
+              <li class="itemAbout"> <a href="about.html">About</a> </li>
+              <li class="itemAbout"> <a href="index.html">Home</a> </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+<main>
+
+<section class="aboutFirstSection">
+  <p class="aboutTitle" href="index.html">TRAFFIC & CONVERSION <br>SUMMIT 2022</p>
+  <p class="aboutDescription">
+    More than 10 hours of training divided into three days in which you will learn from active professionals knowledge, tips and strategies of online advertising and digital strategy.Whether you're starting from scratch or already have experience,
+  </p>
+  <p class="aboutContact">Please contact us by email fot any further questions about Traffic & Conversion Summit - 2022!</p>
+  <p class="aboutEmail">trafficConversion@summit.org</p>
+</section>
+
+<section  class="aboutSecondSection">
+  <p class="about_logo">Traffic & Conversion Summit - 2022 Logo</p>
+        <div class="guideBar"></div>
+        <div class="program_line"></div>
+        <p class="aboutLogo">The logo of Traffic & Conversion Summit - 2022 was decided through the logo competition
+            from  11th to July 29th.
+            <br>After online public voting, the logo from Andreina Barrios won the prize.
+        </p>
+        <img class="logo" src="./src/images/logoBlack.jpg">
+        <p class="about_past">See the past congress of online advertising! </p>
+        <div class="AboutPhotos">
+            <img class="aboutPhoto1" src="./src/images/summit2019.png" alt="TrafficPhoto">
+            <img class="aboutPhoto2" src="./src/images/summit2020.png" alt="TrafficPhoto">
+        </div>
+    </section>
+
+    <section class="aboutSectionPartner">
+        <div class="partners">
+        <p class="aboutPartner">Partners</p>
+        <div class="guideBar"></div>
+        <div class="partnerIcons">
+            <img class="partnerIcon" src="./src/images/mozilla.png">
+            <img class="partnerIcon" src="./src/images/google.png">
+            <img class="partnerIcon" src="./src/images/naver.png">
+            <img class="partnerIcon" src="./src/images/airbnb.png">
+        </div>
+    </div>
+    <div class="license">
+        <h3>Traffic & Conversion Summit</h3>
+        <p class="rigths">2017 congress of online advertising <br> Rigths Reserved</p>
+    </div>
```

Comment:
- [ ] This is a footer hence you should make use of the `<footer>`semantic tag instead of using a `<div>`